apps/web/screens/Search/Search.tsx

```diff
@@ -187,11 +188,20 @@ const Search: Screen<CategoryProps> = ({
   }
 
   const searchResultsItems = (searchResults.items as Array<
-    Article & LifeEventPage & AboutPage & News & AdgerdirPage
+    Article & LifeEventPage & AboutPage & News & AdgerdirPage & SubArticle
   >).map((item) => ({
-    title: item.title,
-    description: item.intro ?? item.seoDescription ?? item.description,
-    link: linkResolver(typenameResolver(item.__typename), [item.slug]),
+    title: item.parent ? item.parent.title + '>' + item.title : item.title,
```

Comment:
NOTE: Here, again, I am testing how the data gets delivered to the front.
This might be unnecessary and not according to the accepted design document.